src/test/java/com/olgaboiar/guacomole/EchoClientTest.java

```diff
@@ -15,11 +16,16 @@
     Socket socket;
 
     @BeforeEach
-    public void setup () throws Exception {
+    public void preTest () throws Exception {
```

Comment:
I'm curious why you choose the word preTest and postTest in favor of beforeEach/setUp and afterEach/tearDown?

Reply:
fixed in https://github.com/olgaboiar/GuacamoleServer/pull/3/commits/33a874d6b5d5f52c7d47e4675a38e82c3c72f35a